webpack doesn't compile with typescript@next (2019-04-09)

I updated lib.dom.d.ts yesterday and thanks to @saschanaz, it now has WebAssembly types. That means that the forward declaration `declare const WebAssembly` in declarations.d.ts needs to be updated to avoid an error in typescript 3.5 and above. I'll put up a PR shortly.

The fix is easy; change the webassembly declaration to `declare namespace WebAssembly {}`.

**What is the current behavior?**

```
$ > tsc
declarations.d.ts:258:15 - error TS2451: Cannot redeclare block-scoped variable 'WebAssembly'.

258 declare const WebAssembly;
                  ~~~~~~~~~~~

  ../../ts/built/local/lib.dom.d.ts:17566:19
    17566 declare namespace WebAssembly {
                            ~~~~~~~~~~~
    'WebAssembly' was also declared here.

../../ts/built/local/lib.dom.d.ts:17566:19 - error TS2451: Cannot redeclare block-scoped variable 'WebAssembly'.

17566 declare namespace WebAssembly {
                        ~~~~~~~~~~~

  declarations.d.ts:258:15
    258 declare const WebAssembly;
                      ~~~~~~~~~~~
    'WebAssembly' was also declared here.


Found 2 errors.
```

**What is the expected behavior?**
No compile error.



**Other relevant information:**
webpack version:  
master

--- a/declarations.d.ts
+++ b/declarations.d.ts
@@ -255,7 +255,7 @@ declare const hotAddUpdateChunk;
 declare const parentHotUpdateCallback;
 declare const $hotChunkFilename$;
 declare const $hotMainFilename$;
-declare const WebAssembly;
+declare namespace WebAssembly {}
 declare const importScripts;
 declare const $crossOriginLoading$;
 declare const chunkId;

